src/lammpsio/data.py

```diff
@@ -394,6 +482,62 @@ def read(self):
                                 "Expected number of columns not read for mass"
                             )
                         masses[int(row[0])] = float(row[1])
+                elif "Bonds" in line:
+                    if N_bonds is not None:
+                        snap.bonds = Bonds(N_bonds)
+                    _readline(f, True)  # blank line
+                    for i in range(int(snap.bonds.N)):
```

Comment:
This should be an `int` already. If it isn't, that needs to be fixed elsewhere
```suggestion
                    for i in range(snap.bonds.N):
```